README.md

```diff
@@ -2,29 +2,54 @@
 
 [![CircleCI Build Status](https://circleci.com/gh/all-of-us/workbench.svg)](https://circleci.com/gh/all-of-us/workflows/workbench)
 
-## Getting the code
+## Setup for Development
 
-* Run `git clone https://github.com/all-of-us/workbench`
+System requirements:
+
+  * [Docker CE](https://www.docker.com/community-edition)
+  * [Ruby](https://www.ruby-lang.org/en/downloads/)
+  * [Python](https://www.python.org/downloads/) >= 2.7.9
+  * [gcloud](https://cloud.google.com/sdk/docs/#install_the_latest_cloud_tools_version_cloudsdk_current_version)
```

Comment:
Note that I consider the Python and gcloud requirements to be bugs. Having docker should take care of both of these.